pkg/kv/kvserver/protectedts/ptpb/protectedts.proto

```diff
@@ -119,8 +132,18 @@ message Record {
   // change the Version of the subsystem.
   bool verified = 6;
   
-  // Spans are the spans which this Record protects.
-  repeated roachpb.Span spans = 7 [(gogoproto.nullable) = false];
+  // DeprecatedSpans are the spans which this Record protects.
+  repeated roachpb.Span deprecated_spans = 7 [(gogoproto.nullable) = false];
+
+  // Target holds information about the objects on which the protected timestamp
+  // record applies.
+  oneof target {
+    SchemaObjectsTarget schema_objects = 8;
+    TenantsTarget tenants = 9;
+    bool cluster = 10;
```

Comment:
This might be easier to represent with a standalone ClusterTarget type with its own commentary. It's unclear what it means if the boolean is set one way or another, it's the fact that it's non-nil that we care about with these union types.

Reply:
heh you beat me to it, just making this change.